src/components/Donate.jsx

```diff
@@ -0,0 +1,59 @@
+// MyStoreCheckout.js
+import React, { Component } from 'react';
+import AsciiOdie from './AsciiOdie'
+import Loading from './Loading'
+import DonateForm from './DonateForm'
+import {StripeProvider} from 'react-stripe-elements';
+import {Elements} from 'react-stripe-elements';
+import {StripeKeys} from '../utilities/stripeKeys';
+
+class Donate extends Component<{}, {stripe: null | StripeShape}> {
+  constructor(props) {
+    super(props);
+
+    this.stripeKey = StripeKeys.test;
+
+    this.state = {
+      stripe: null,
+    };
+  }
+
+  componentDidMount() {
+    // inject Stripe.js
```

Comment:
Is this ever removed from the DOM?

Reply:
Stripe recommends keeping it on all pages. We might want to inject it even earlier.